src/Illuminate/Console/Process/PendingProcess.php

```diff
@@ -0,0 +1,74 @@
+<?php
+
+namespace Illuminate\Console\Process;
+
+use Illuminate\Support\Arr;
+use Illuminate\Support\Traits\Macroable;
+use Symfony\Component\Process\Process;
+
+class PendingProcess
+{
+    use Macroable;
+
+    /**
+     * The process's arguments.
+     *
+     * @var iterable<array-key, string>
+     */
+    protected $arguments;
+
+    /**
+     * The process's path.
+     *
+     * @var string|null
+     */
+    protected $path;
+
+    /**
+     * Create a new Pending Process instance.
+     *
+     *
+     * @return void
+     */
+    public function __construct()
+    {
+        // ..
```

Comment:
I don't know any other class in Laravel which have an empty constructor. I am not against it as the docs are useful, but I wonder about consistency.

Also doc block has 2 empty lines instead of one.

Reply:
This PR is a work in progress, ignore the code for now.